bigbluebutton-html5/imports/api/voice-users/server/methods/muteToggle.js

```diff
@@ -17,5 +21,16 @@ export default function muteToggle(credentials, userId) {
     mutedBy: requesterUserId,
   };
 
+  const { role: requesterRole } = Users.findOne({
+    meetingId,
+    userId: requesterUserId,
+  });
+
+  const { muted: isUserMuted } = VoiceUsers.findOne({
```

Comment:
If there is no such a voice user - it will return `null`. ([last sentence](https://docs.mongodb.com/manual/reference/method/db.collection.findOne/#db.collection.findOne)
But you are trying to extract `muted` from `null` - it will fail here.
Same for the `role` above.